src/network/api/homePageApi.ts

```diff
@@ -37,6 +37,39 @@ export const getCanonizedNameSpacesApi = async () => {
   }
 };
 
+export const getRecentActivitiesApi = async (reqBody) => {
+  try {
+    // const recentActivities = await NetworkCall.fetch(
+    //   HomePageRequests.getCanonizedRecentActivities(reqBody),
+    //   false
+    // );
+    const mockData = [
+      {
+        id: 27,
+        log_name: "threads",
+        description: "A thread has been updated",
+        subject_type: "App\\Models\\Languages",
+        subject_id: 20,
+        causer_type: null,
+        causer_id: null,
+        properties: [],
+        created_at: "2022-02-17T11:22:36.000000Z",
```

Comment:
I think we need to discuss at it because we are using unix timestamp across systems.

Reply:
Will be replaced with API's response